README.md

```diff
@@ -4,19 +4,30 @@ go-algorand
 ====================
 Algorand's official implementation in Go.
 
-Algorand is a permissionless, pure proof-of-stake blockchain that delivers decentralization, scalability, security, and transaction finality.
+Algorand is a permissionless, pure proof-of-stake blockchain that delivers
+decentralization, scalability, security, and transaction finality.
 
 ## Getting Started ##
 
-Our [developer website][developer site url] has the most up to date information about using and installing the algorand platform.
+Our [developer website][developer site url] has the most up to date information
+about using and installing the algorand platform.
 
 ## Building from source ##
 
-Development is done using the [Go Programming Language](https://golang.org/) version 1.12.x, and this document assumes that you have a functioning environment setup. If you need assistance setting up an environment please visit the [official Go documentation website](https://golang.org/doc/).
+Development is done using the [Go Programming Language](https://golang.org/)
+version 1.12.x, and this document assumes that you have a functioning
+environment setup. If you need assistance setting up an environment please visit
+the [official Go documentation website](https://golang.org/doc/).
 
 ### Linux / OSX ###
 
-We currently strive to support Debian based distributions with Ubuntu 18.04 being our official release target. Our core engineering team uses Linux and OSX, so both environments are well supported for development.
+We currently strive to support Debian based distributions with Ubuntu 18.04
+being our official release target. Our core engineering team uses Linux and OSX,
+so both environments are well supported for development.
+
+[Homebrew (brew)](https://brew.sh) must be installed before
```

Comment:
Could you mention this requirement applies only for MacOS builds ? on ubuntu we use apt-get.

Reply:
added "OSX only" note.